rs/Cargo.toml

```diff
@@ -23,6 +23,8 @@ rand = "0.8.5"
 russh = { version = "0.37.1", default-features = false, features = ["openssl", "flate2"], optional = true }
 russh-keys = { version = "0.37.1", default-features = false, features = ["openssl"], optional = true }
 hyper = "0.14"
+os_info = { version = "3", default-features = false }
+winreg = "0.8"
```

Comment:
This only needs to be a dependency on Windows https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#platform-specific-dependencies

Reply:
Added that to the `Cargo.toml` file for just windows